### DISCUSSION TITLE
problem after restart

### DISCUSSION THREAD

**Discussion by rieders** [inventree/InvenTree#5633] [General]
Hello
Today I had to restart my instance of inventree. Now all I see is the Apache2 Debian Default Page window. I had previously created a new item and created a barcode. When I scanned the barcode, the article wasn't called up.
How can I access the Inventree interface again?

Greetings Andre

**Comment by matmair** [comment]
We will need more information
1) what OS are you running on
2) which deployment method
3) do you roughly know the version or when you installed / updated it
4) did you recently install OS updates
5) do you have other websites running


By default we are recommending nginx so apache2 could have superseded nginx. You could try to stop apache2 'sudo service apache2 stop' and start nginx 'sudo service nginx start'

**Comment by rieders** [comment]
Hello. 

Thank you for the quick help. I disabled apache and enabled nginx. now I can access it again. how can I make nginx the default web server?

I still have another question. some plugins like KiCad have an incorrect path (plugin/inventree-kom2/.) which means the link to the plugin is not correct. Where do I have to change that?

Greetings and thank you again.
